apps/site/pages/docs/getting-started/2-setting-up-the-project.mdx

```diff
@@ -89,44 +91,49 @@ Before cloning your store's repository, ensure that you have the following tools
   Although you might opt to use an alternative code editor, note that Visual Studio Code might help you follow along with the FastStore documentation since our documentation may include screenshots from VS Code.
 </Callout>
 
+---
+
 ## Step 1: Cloning the Store's Repository
-1. Open the terminal and navigate to the directory where you want to store your project.
+
+### 1. Open the terminal and navigate to the directory where you want to store your project.
```

Comment:
I was checking this doc, and it seems a bit off the steps as `h3` 🤔 
![example](https://github.com/vtex/faststore/assets/67270558/afa4fff1-cc24-4bf6-b05c-eb6a7e9ae8f1)

I suggest we leave it as a numbered list as before.